src/webapp/views.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import permissions
+from rest_framework.schemas import get_schema_view
+from rest_framework_json_api.schemas.openapi import SchemaGenerator
+
+schema_view = get_schema_view(
+    title="<NAME THE SCHEMA>",
```

Comment:
@km1230 We either need to document that this field needs to be set in the README.md or (and this is my preference) set the title to be `settings.PROJECT_NAME`